### PR TITLE
Add bbc entry in lit tests

### PR DESCRIPTION
**Pull request by Rin18**

### CODE REVIEW

**Review by kiranchandramohan — APPROVED** [review]
LGTM.

File: flang/test/lit.cfg.py [hunk 1]
```diff
@@ -88,6 +88,9 @@
 # the build directory holding that tool.
 tools = [
   ToolSubst('%f18', command=FindTool('f18'),
+    extra_args=["-intrinsic-module-directory "+config.flang_intrinsic_modules_dir],
+    unresolved='fatal'),
+    ToolSubst('%bbc', command=FindTool('bbc'),
```
Comment (schweitzpgi) [inline flang/test/lit.cfg.py:93]: Looks like something happened with the indentation.

Comment (Rin18) [reply flang/test/lit.cfg.py:93]: I think I followed the same indentation as for the f18 entry. It looks fine to me? Am I missing something?

Comment (schweitzpgi) [reply flang/test/lit.cfg.py:93]: The first ToolSubst is indented 2 spaces less than this one on my screen.

Comment (Rin18) [reply flang/test/lit.cfg.py:93]: You're right. I'll fix this right away.

**Review by schweitzpgi — APPROVED** [review]
I'd prefer to have a separable explicit substitution to include the config setting of the intrinsic module directory so the tests could use something like `%bbc %imoddir` instead of requiring the passing of a new command line setting to all tests. The option is after all unneeded by all of the current tests.

On the other hand, this seems ok and low risk.

**Pull request merged**